components/cam/src/physics/cam/restart_physics.F90

```diff
@@ -20,7 +20,7 @@ module restart_physics
   use cospsimulator_intr, only: docosp
   use radiation,          only: cosp_cnt_init, cosp_cnt, rad_randn_seedrst, kiss_seed_num
   use physics_types,      only: physics_state
-  use dyn_grid,           only: fv_nphys
+  ! use dyn_grid,           only: fv_nphys
```

Comment:
Did you comment out the large chucks of code block in restart_physcis.f90 because you are planning to revisit them later?

Reply:
oh, I meant to remove these lines, I'll do that.